gems/bitcoin_vanity/CVE-2019-15224.yml

```
@@ -0,0 +1,16 @@
+---
+gem: bitcoin_vanity
+cve: 2019-15224
+ghsa: 333g-rpr4-7hxq
+url: https://github.com/rubygems.org/wiki/Gems-yanked-and-accounts-locked#19-aug-2019
+date: 2019-08-20
+title: Critical severity vulnerability that affects bitcoin_vanity
+description: |-
+  The bitcoin_vanity gem 4.3.3 for Ruby, as distributed on RubyGems.org, included a code-execution backdoor inserted by a third party.
+  No unaffected version is known to exist, as the gem appears to have been entirely removed.
+unaffected_versions:
+  - "< 4.3.3"
+  - "> 4.3.3"
+related:
+  url:
+  - https://github.com/rubygems.org/issues/2097
```

I believe this URL is incorrect and should be: 

```suggestion
  - https://github.com/rubygems/rubygems.org/issues/2097
```

This incorrect URL is repeated through out the PR.